packages/packagers/js/src/DevPackager.js

```diff
@@ -162,6 +164,7 @@ export class DevPackager {
       ) +
       ', ' +
       JSON.stringify(this.parcelRequireName) +
+      (globalVar ? ', ' + JSON.stringify(globalVar) : '') +
```

Comment:
You are already checking on `globalVar` on line (151) and you are checking it again here?